### PR TITLE
Install addons-validator and configure it's path. Fixes #881

### LINKED ISSUE #881: Integrate addons-validator into olympia build
Olympia part to https://github.com/mozilla/addons-validator/issues/62

* [x] Add as dependency in package.json (https://www.npmjs.com/package/addons-validator)
* [x] Update paths to find and execute the binary

### PR DESCRIPTION
**Pull request by EnTeQuAk**
Pleae do not merge yet, the addons-validator requires a nodejs version that's not present in our current docker / production systems.

This depends on all the work going on in https://github.com/mozilla/olympia/pull/963

### CODE REVIEW

**Comment by EnTeQuAk** [comment]
The failing travis test is from my accidentally pushed branch to olympia repo directly.

File: settings.py [hunk 1]
```diff
@@ -54,10 +54,15 @@
 # will be the path to the `stylus` and `lessc` executables.
 STYLUS_BIN = os.getenv('STYLUS_BIN', path('node_modules/stylus/bin/stylus'))
 LESS_BIN = os.getenv('LESS_BIN', path('node_modules/less/bin/lessc'))
-CLEANCSS_BIN = os.getenv('CLEANCSS_BIN',
-                         path('node_modules/clean-css/bin/cleancss'))
-UGLIFY_BIN = os.getenv('UGLIFY_BIN',
-                       path('node_modules/uglify-js/bin/uglifyjs'))
+CLEANCSS_BIN = os.getenv(
+    'CLEANCSS_BIN',
+    path('node_modules/clean-css/bin/cleancss'))
```
Comment (magopian) [inline settings.py]: We'll have to discuss this indentation sometimes ;)

File: Dockerfile [hunk 2]
```diff
@@ -61,3 +61,4 @@ ENV CLEANCSS_BIN /srv/olympia-node/node_modules/clean-css/bin/cleancss
 ENV LESS_BIN /srv/olympia-node/node_modules/less/bin/lessc
 ENV STYLUS_BIN /srv/olympia-node/node_modules/stylus/bin/stylus
 ENV UGLIFY_BIN /srv/olympia-node/node_modules/uglify-js/bin/uglifyjs
+ENV VALIDATOR_BIN /srv/olympia-node/node_modules/addons-validator/addons-validator
```
Comment (magopian) [inline Dockerfile]: is it missing a `bin/` in there?

**Comment by magopian** [comment]
r+wc. Such excitement, wow addons-validator, much happiness!

File: settings.py (hunk 1 above)
Comment (andymckay) [inline settings.py]: By that you mean, this is the one and true indentation and all other forms are wrong? :smiling_imp:

Comment (magopian) [inline settings.py]: I was going to say the opposite, I'm not sure this indentation is the best. It's not like it matters that much, but wonder if we can just agree on a set of rules, and then maybe "enforce" it (using flake8 or pyformat or anything like that) so we don't have to wonder each time if it's coherent with the rest of the code or not ;)

Comment (EnTeQuAk) [inline settings.py]: Andy, <3

I think that might be a :beers:-topic for Mozlando :smile:

Comment (andymckay) [inline settings.py]: In my mind:

```
CLEANCSS_BIN = os.getenv(
    'CLEANCSS_BIN',
    path('node_modules/clean-css/bin/cleancss'))
```
Requires no work if you change CLEANCSS_BIN. If you change CLEANCSS_BIN:
```
OMG_WHY_DO_YOU_HATE_ME_PEP8 = os.getenv('CLEANCSS_BIN',
                         path('node_modules/clean-css/bin/cleancss'))
```
Welcome to a whole pile of code that now fails pep8 and more work.

I hate work.

Comment (magopian) [inline settings.py]: This is maybe the best explanation/reason i've seen. You just convinced me.

So now, let's flake8/enforce this thing (one day, maybe).

**Pull request merged**